test/e2e/framework/expect.go

```diff
@@ -20,6 +20,16 @@ import (
 	"github.com/onsi/gomega"
 )
 
+// ExpectBeNil expects actual is equivalent to gomega.BeNil()
```

Comment:
I know we discussed this function locally in advance, but I have a question now.
Can we use `ExpectEqual(actual, nil)` instead of adding a new function?
That can be more clear if possible instead of `gomega.BeNil'.

Reply:
agree on not adding more helper functions than we actually need

Reply:
@liggitt . Sure. i get it . Thanks